schemas/json/component/list-component.schema.v1.json

```diff
@@ -0,0 +1,115 @@
+{
+  "$id": "https://altinncdn.no/schemas/json/component/list-component.schema.v1.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "List component",
```

Comment:
`List` seems to be missing `dataModelBindings`. In this case, there really are no fixed binding keys, it is up to the developer to select which columns to store:
https://docs.altinn.studio/app/development/ux/components/listcomponent/#how-to-store-data-in-the-data-model

Reply:
Added `dataModelBindings` for `List`, with no set property names.